.github/workflows/pull_requests.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Scan for Vulnerabilities in Code
         uses: Templum/govulncheck-action@v1.0.0
         with:
-          go-version: '1.21'
+          go-version: '1.22'
           package: ./...
           fail-on-vuln: false
```

Comment:
if this resolves the govulncheck error then we can re-enable this check

Reply:
Are you sure this is currently disabled ?

Reply:
Yes, it's disabled now